utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/serde/CirceJsonDeserializer.scala

```diff
@@ -28,7 +28,8 @@ class CirceJsonDeserializer(jsonSchema: Schema) {
     val jsonObject = new JSONTokener(string).nextValue()
     //after validate jsonObject has set default field values
     catchValidationError(jsonSchema.validate(jsonObject))
-    JsonToNuStruct(toCirce(jsonObject), swaggerTyped)
+    val struct = JsonToNuStruct(toCirce(jsonObject), swaggerTyped)
+    struct
```

Comment:
let's leave it as it was before

Reply:
Hmm, I would rather leave it.. But if you think so..

Reply:
No it's ok. I tought that it was by mistake.